core/domain/app_feedback_report_domain.py

```diff
@@ -463,7 +463,7 @@ def to_dict(self) -> Dict[str, Any]:
         """
         return {
             'report_type': self.report_type.name,
```

Comment:
Should this one also be updated to use value? What about other cases in this file & throughout the codebase? I am still a bit concerned about the inconsistency as it may cause confusion among other developers on the team, too.

Reply:
I'm not very sure, sorry. However, I've noticed that @vojtechjelinek primarily reviews other PRs of this kind.

@vojtechjelinek could you PTAL when you find the time? For reference, here's [the PR which changed the formatting of the report_type class](https://github.com/oppia/oppia/pull/15296) -- it changed some instances of the enum to use the `.value` property, and left some with the `.name` property. As Ben said, it's inconsistent, but I'm not very sure why some instances of this were left unchanged so I guess there must be some good reason for it?

Reply:
Yeah, it should be `value` everywhere, not sure why it wasn't all changed before.

Reply:
Thanks for checking @nithinrdy, and thanks for confirming @vojtechjelinek!